test/node/update-api.js

```diff
@@ -55,3 +90,79 @@ test.test('must update API', (test) => {
     }
   );
 });
+
+test.test(
+  'must not update API on existing connection if no supported version is found',
```

Comment:
We should clarify test description here and in the tests after this one (probably adding 'version' should be enough), it'll be hard to understand the problem if one of those fails.

Reply:
@lundibundi could you please clarify proposed change?

Reply:
@nechaido We can add version to the description ''must not update API on existing connection if no supported version is found (connect to @1)" and so with other tests in order to clearly discern them should an error occur.